testsuite/testsuite-keycloak-mpjwt/pom.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2015 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.thorntail.testsuite</groupId>
+    <artifactId>testsuite-parent</artifactId>
+    <version>2.3.0.Final-SNAPSHOT</version>
+    <relativePath>../</relativePath>
+  </parent>
+
+  <artifactId>testsuite-keycloak-mpjwt</artifactId>
+
+  <name>Test Suite: Keycloak MP JWT</name>
+  <description>Test Suite: Keycloak MP JWT</description>
+
+  <packaging>jar</packaging>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.thorntail</groupId>
+      <artifactId>microprofile-jwt</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.keycloak</groupId>
+      <artifactId>keycloak-core</artifactId>
+      <version>4.8.1.Final</version>
```

Comment:
There should be a constant for this somewhere. We'll be updating to the latest 4.8.x soon (currently, x == 3, hopefully that won't change).

Reply:
Thanks. testsuites don't inherit from the build-parent, but I will push it to the testsuite-parent. However I'm still not sure where this KC factory should actually reside in the main source code. So it is also likely these direct keycloak dependencies will go away from this test :-)